kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -191,6 +191,11 @@ data:
     # [agent_settings.proxy_config]
     #    ignore_proxy_settings = "true"  # if this is not applied, default value is false
 
+    # Uses fluent-bit for perf and container inventory instead of fluentd
+    # Disables custom metrics
+    # [agent_settings.resource_optimization]
+    #    enabled = "false"
```

Comment:
does this work, if customer has enabled with boolean flag  true/false? we have inconsitency on the data type of enabled flag.